cmd/containerd/command/notify_systemd.go

```diff
@@ -0,0 +1,47 @@
+// +build linux,!no_systemd
```

Comment:
Rename 
notify_linux.go => notify_no_systemd.go
notify_systemd.go => notify_linux.go
I think the Commit is clearer like this

Reply:
> Rename
> notify_linux.go => notify_no_systemd.go
> notify_systemd.go => notify_linux.go
> I think the Commit is clearer like this

I don't think so, because on *Linux* there just is no such thing like service notify - it's a *very special systemd* thing,

Actually, it would be more clear if it was an opt-in (explicitly set 'systemd' tag), but got complaints about downstreams
too lazy to add the tag when they really wanna have systemd stuff enabled.

Reply:
Please rename, as the default config supports systemd.